### PR TITLE
Add several spelling corrections + refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6926,6 +6942,8 @@ compeltly->completely
 compenent->component, competent,
 compenents->components, competence,
 compensantion->compensation
+compenstate->compensate
+compenstated->compensated
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:6946]:
```text
compenstates->compensates
```
https://github.com/search?q=compenstates&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11730,6 +11756,7 @@ distribtions->distributions
 distribtuion->distribution
 distribtuions->distributions
 distribtution->distributions
+distribue->distribute
 distribued->distributed
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11760]:
```text
distribues->distributes
```
https://github.com/search?q=distribues&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -34555,6 +34623,7 @@ unpacket->unpacked
 unparseable->unparsable
 unpertubated->unperturbed
 unperturbated->unperturbed
+unperturbe->unperturb, unperturbed,
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Thanks. In that case, I added `unperturb->unperturbed` as a separate entry

**Comment by luzpaz** [comment]
@vikivivi added your mods, rebased. Please re-review. TIA!

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**